src/modules/messages/proxyPluralMessage.ts

```diff
@@ -37,16 +43,26 @@ export const proxyPluralMessage = async (
       allowedMentions: {
         parse: [],
       },
+      embeds: [],
     };
 
     if (message.type === MessageType.Reply && message?.reference?.messageId) {
       const originalMsg = await message.channel.messages.fetch(
         message.reference.messageId
       );
-      originalMsg.reply(content);
-    } else {
-      await webhook.send(content);
+
+      content.content =
+        (message.mentions.users.size ? `<@${originalMsg.author.id}>, ` : "") +
+        `${content.content}`;
```

Comment:
```suggestion
      content.content =
        (message.mentions.users.size ? `<@${originalMsg.author.id}>, ` : "") +
        `${content.content}`;
      content.allowedMentions = {
        users: [originalMsg.author.id],
      };
```

Since it starts empty, we need to explicitly set this or the ping won't work.